website/docs/plugin/sdkv2/resources/index.mdx

```diff
@@ -30,3 +30,12 @@ Terraform has data consistency rules for resources, which may not be easily disc
 Resources define the data types and API interactions required to create, update, and destroy infrastructure with a cloud vendor, while the [Terraform state](/terraform/language/state) stores mapping and metadata information for those remote objects.
 
 When resource implementations change (due to bug fixes, improvements, or changes to the backend APIs Terraform interacts with), they can sometimes become incompatible with existing state. When this happens, a migration is needed for resources provisioned in the wild with old schema configurations. Terraform resources support migrating state values in these scenarios via [State Migration](/terraform/plugin/sdkv2/resources/state-migration).
+
+## Write-only Attributes
```

Comment:
Commented on this above, but suggest renaming everywhere to  "Write-only arguments" for consistency!